Makefile

```diff
@@ -49,44 +39,43 @@ deploy:
 update: build deploy
 
 test:
-	@docker run --rm --name postgres -e POSTGRES_PASSWORD=test1234 -p 5432:5432 -d postgres:11-alpine
-	@docker run --rm --name mysql -e MYSQL_ROOT_PASSWORD=test1234 -p 3306:3306 -d mysql:5.7
+	@docker-compose up -d
+	@docker-compose restart sqladmin
```

Comment:
Why you need to restart?

Reply:
sqladmin has small json database which basically writes a file in filesystem to keep track of created instances.
to have fresh new test, it has to be clean otherwise, the next test result will be effected